app/views/works/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<section>
+  <h2><%= @work.title %> </h2>
+  <p>Created by: <%= @work.creator %> </p>
+  <p>Published: <%= @work.publication_year %> </p>
+  <p><%= @work.description %></p>
+</section>
+<section>
+    <span><%= button_to "Back to media ranks", root_path, :class => "btn btn-primary", form: {style: 'display:inline-block;'}%></span>
+    <span><%= button_to "Edit", edit_work_path, :class => "btn btn-primary", form: {style: 'display:inline-block;'}%></span>
```

Comment:
I think both of these buttons are trying to use the `post` verb, which is breaking the routes!